placementApp/models.py

```diff
@@ -1,3 +1,111 @@
 from django.db import models
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
+from django.core.validators import RegexValidator
 
-# Create your models here.
+
+class MyAccountManager(BaseUserManager):
+	def create_user(self, email, username, password=None):
+		if not email:
+			raise ValueError('Users must have an email address')
+		if not username:
+			raise ValueError('Users must have a username')
+
+		user = self.model(
+			email=self.normalize_email(email),
+			username=username,
+		)
+
+		user.set_password(password)
+		user.save(using=self._db)
+		return user
+
+	def create_superuser(self, email, username, password):
+		user = self.create_user(
+			email=self.normalize_email(email),
+			password=password,
+			username=username,
+		)
+		user.is_admin = True
+		user.is_staff = True
+		user.is_superuser = True
+		user.save(using=self._db)
+		return user
+
+
+class User(AbstractBaseUser):
+	email = models.EmailField(verbose_name="email", max_length=60, unique=True)
+	username = models.CharField(max_length=30, unique=True)
+	date_joined = models.DateTimeField(verbose_name="date joined", auto_now_add=True)
+	last_login = models.DateTimeField(verbose_name="last login", auto_now=True)
+	is_admin = models.BooleanField(default=False)
+	is_active = models.BooleanField(default=True)
+	is_staff = models.BooleanField(default=False)
+	is_superuser = models.BooleanField(default=False)
+	ROLE_CHOICES = [("STUDENT", "Student"), ("CO", "Co-ordinator"), ("TPO", "TPO")]
+	role = models.CharField(max_length=7, blank=False, choices=ROLE_CHOICES)
+
```

Comment:
There is no need to keep is_admin, is_staff. is_superuser since we already have role field